### ISSUE TITLE
Steam Link support please!

### ISSUE THREAD

**Issue by deblueseven** [X2CommunityCore/xcom2-launcher#42]
I can't play this game without this launcher and steam link dosen't support it or something. My controller setup just won't be detected for XCOM 2. The controller acts as if it's still in desktop mode... Please, I can't play with mouse and keyboard and I'd really like to use my Steam Link in my living room.

**Comment by robojumper** [comment]
I don't think this is something we can fix on our side. I had the same problem, and used this workaround:

1. Find the XCOM 2 Executable: Depending on whether you want to play Vanilla or WotC, it's `"PATH_TO_STEAMAPPS\common\XCOM 2\Binaries\Win64\XCom2.exe"` or `"PATH_TO_STEAMAPPS\common\XCOM 2\XCom2-WarOfTheChosen\Binaries\Win64\XCom2.exe"`
2. In your library, click "Add A Game..." -> "Add a Non-Steam-Game..." ->"Browse..." and select the executable, then click "Add selected programs".
3. Right-click the new shortcut -> "Properties" -> "Set Launch Options..." and paste `-fromlauncher -noredscreens -review`, and confirm.

Now you can use this launcher to choose mods, and the steam shortcut you created to play XCOM 2. This worked for me with my Steam Link and a XBOX 360 controller.

**Comment by deblueseven** [comment]
God damn it, I appreciate the support but it did not work... I get big picture overlay as always but my xcom 2 controller settings does not get recognized. Instead it uses desktop mode. Also when I go into control configuration in the game the controller literally loses function and I have to alt, f4... I did exactly as you said, I have my profile set up in the alt launcher, 114 mods. And yet, launching the game theo6thos new shortcut in steam launches WOTC without any mods whatsoever so there are numerous issues here :( my steam link is useless because I HAVE to use this launcher to be able to play with my mods lest I get immediate CTD.

**Comment by jangerhard** [comment]
If anyone else are having these issues still, there is a workaround.

Add the following to the launch-options for XCOM 2 in steam: 
`"{PATH_TO_AML}\XCOM2 Launcher.exe" %command%`
